tests/integration/conftest.py

```diff
@@ -1,10 +1,10 @@
 import os
-
 import address_book as ab
 import boa
 import pytest
 
 boa.set_etherscan(api_key=os.getenv("ETHERSCAN_API_KEY"))
+BOA_CACHE = False
```

Comment:
Can't we enable this by default since we will be restricting the fixtures scope?

Reply:
nope, can't :(